### PR TITLE
COMMERCE-10175

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://issues.liferay.com/browse/COMMERCE-10175

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce

**Comment by Brunno0liveira** [comment]
ci:test:commerce

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCheckout.testcase [hunk 1]
```diff
@@ -0,0 +1,241 @@
+@component-name = "portal-commerce"
+definition {
+
+	property custom.properties = "jsonws.web.service.paths.excludes=";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property release.feature.flags.disable.DISABLE_PRIVATE_LAYOUTS = "true";
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCheckout.testcase]: This property can be removed as I'm not seeing any usage of the private pages on the tests

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCheckout.testcase [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+		}
+
+		task ("And the tier list are changed") {
+			CommerceNavigator.gotoPortlet(
+				category = "Pricing",
+				portlet = "Price Lists");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCheckout.testcase]: Here you can use `CommercePriceLists.openPriceListsAdmin();` instead

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCheckout.testcase [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+				locator1 = "TextInput#ANY",
+				value1 = "50.00");
+
+			Click(locator1 = "Button#SUBMIT");
+
+			SelectFrameTop();
+
+			SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
+
+			Button.clickSave();
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCheckout.testcase]: Can use the macro instead: CommercePriceLists.createPriceListPriceTiers().

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by lorenzolapescara** [comment]
Hi @Brunno0liveira, I've reviewed the tests, please have a look at the comments I left.
Please remember to change the property portal.acceptance.
Thanks, Lorenzo.

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by victoroscuro** [comment]
ci:test:commerce

**Comment by victoroscuro** [comment]
ci:stop

**Pull request closed without merging**